src/environments/environment.ts

```diff
@@ -3,6 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
+  firebase: {
```

Comment:
I suggest that we create file firebase.config.ts` that exports `FirebaseConfig` and inject it here. maybe we can encrypt that file and push it to git? I am still not sure on how to handle best way those credentials